packages/lab/src/slider/internal/SliderThumb.tsx

```diff
@@ -0,0 +1,63 @@
+import { makePrefixer, Label } from "@salt-ds/core";
+import { clsx } from "clsx";
+import { getPercentage } from "./utils";
+import { ComponentPropsWithoutRef, RefObject } from "react";
+import { useMouseDownThumb } from "./useMouseDownThumb";
+import { useKeyDownThumb } from "./useKeyDownThumb";
+import { useSliderContext } from "./SliderContext";
+
+const withBaseName = makePrefixer("saltSliderThumb");
+
+export interface SliderThumbProps extends ComponentPropsWithoutRef<"div"> {
+  trackRef: RefObject<HTMLDivElement>;
+}
+
+export function SliderThumb(props: SliderThumbProps): JSX.Element {
+  const { trackRef, className, ...rest } = props;
+
+  const { min, max, step, value, setValue, onChange, ariaLabel } =
+    useSliderContext();
+
+  const onKeyDown = useKeyDownThumb(min, max, step, value, setValue, onChange);
+
+  const { thumbProps, thumbFocus } = useMouseDownThumb(
+    trackRef,
+    min,
+    max,
+    step,
+    value,
+    setValue,
+    onChange
+  );
+
+  const percentage = getPercentage(min, max, value);
+
+  return (
+    <div
+      style={{ left: `${percentage}` }}
```

Comment:
positioning done using absolute positioning, using percentage values from the left side of the track